Makefile

```diff
@@ -62,7 +62,8 @@ IMAGE_NAME := $(BIN)
 
 IMAGE := $(REGISTRY)/$(IMAGE_NAME)
 
-BUILD_IMAGE ?= ghcr.io/kanisterio/build:v0.0.23
```

Comment:
@mabhi You can use this github workflow to build and push a new build image. I believe we can run it once we merge the changes to the Dockerfile

Reply:
Sure @pavannd1 , I actually was looking for this. For the time being I tested with a docker image to validate. Will push a new commit after running the workflow.